app/src/main/java/com/celzero/bravedns/ui/HomeScreenFragment.kt

```diff
@@ -710,10 +770,10 @@ class HomeScreenFragment : Fragment(R.layout.fragment_home_screen) {
             b.fhsDnsOnOffBtn.setBackgroundResource(R.drawable.rounded_corners_button_accent)
             b.fhsDnsOnOffBtn.text = getString(R.string.hsf_stop_btn_state)
             //b.fhsDnsOnOffBtn.setTextColor(fetchTextColor(R.color.black_white))
-            when(braveMode){
-                0 -> b.fhsAppConnectedDesc.text = getString(R.string.dns_explanation_dns_connected)
-                1 -> b.fhsAppConnectedDesc.text = getString(R.string.dns_explanation_firewall_connected)
-                else ->  b.fhsAppConnectedDesc.text = getString(R.string.dns_explanation_connected)
+            when (braveMode) {
+                Constants.APP_MODE_DNS -> b.fhsAppConnectedDesc.text = getString(R.string.dns_explanation_dns_connected)
+                Constants.APP_MODE_FIREWALL -> b.fhsAppConnectedDesc.text = getString(R.string.dns_explanation_firewall_connected)
+                else -> b.fhsAppConnectedDesc.text = getString(R.string.dns_explanation_connected)
```

Comment:
this code-block is repeated at least in three places in this file alone, and is repeated in other files too. abstract it out, and use a common method.

Reply:
Abstracted it to a common method.